modules/cart/src/Plugin/Block/CartBlock.php

```diff
@@ -74,7 +75,7 @@ public static function create(ContainerInterface $container, array $configuratio
    */
   public function defaultConfiguration() {
     return [
-      'item_text' => $this->t('items'),
+      'item_text' => 'item' . PluralTranslatableMarkup::DELIMITER . 'items',
```

Comment:
@bojanz I asked around, there is no standard way, especially with config. I think this is good to go.